.travis.yml

```diff
@@ -1,9 +1,8 @@
 language: python
 python:
-  - 2.7
   - 3.5
   - 3.6
-
+  - 3.8
```

Comment:
NIT: 3.7 maybe too? But I doubt that its really gonna fail on just that version :P